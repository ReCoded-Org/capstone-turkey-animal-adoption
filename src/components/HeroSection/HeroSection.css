.container {
    background-position: center;
    background-size: cover;
    height:55vh;
    width:100%;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    
}

.findmebutton {
    width: 100%;
    max-width:200px;
    background:#FF7777;
    color:white;
    border-radius: 25px;
    border:none;
    outline: none;
    padding: 0.5rem 0;
}
.container h2 {
    color: #FF3E3E;
}
.container p {
    color: #777777;
    
}
.buttonlogo {
    width: 100%;
    max-width:50px;

}

.row {
align-content: flex-start;
}

