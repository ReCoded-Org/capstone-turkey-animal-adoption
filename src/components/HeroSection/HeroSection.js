import React from "react";
import { Container, Row, Col } from "react-bootstrap";
import logo from "../../images/buttonlogo.png";
import "./HeroSection.css"

const HeroSection = (props) => {
    console.log(props)
    const background ={
            backgroundImage: `url(${props.heroSectionMaterial.img})`,

           
          };
    

    
      
    return(
        <Container className="container" style={background}

      >
          <Row className="row">
            <Col sm="8">
              <h2>{props.heroSectionMaterial.title}</h2>
              <p>{props.heroSectionMaterial.text}</p>
              <a href="/search"><button className="findmebutton"><img className="buttonlogo" src={logo}/>Find A Pet</button></a>
            </Col>
          </Row>
        </Container>
    )
    };

export default HeroSection;