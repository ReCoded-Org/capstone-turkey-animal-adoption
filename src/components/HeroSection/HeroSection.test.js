import HeroSection from "./HeroSection.js";
import React from "react";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<HeroSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
