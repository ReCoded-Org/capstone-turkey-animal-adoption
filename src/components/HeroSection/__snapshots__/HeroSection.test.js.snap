// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="containerForLeft container"
  style={
    Object {
      "backgroundImage": "url(https://ggsc.s3.amazonaws.com/images/uploads/The_Science-Backed_Benefits_of_Being_a_Dog_Owner.jpg)",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-sm-8"
    >
      <h2>
        Adopt Please
      </h2>
      <p>
        We are non-profit organization
      </p>
      <a
        href="/search"
      >
        <button
          className="findMeButton"
        >
          <img
            className="buttonLogo"
            src="buttonlogo.png"
          />
          heroSectionHome.button
        </button>
      </a>
    </div>
  </div>
</div>
`;
